docs/releases/5.0.md

```diff
@@ -89,6 +91,37 @@ The following features deprecated in Wagtail 4.0 have been fully removed. See [W
 
 The undocumented `Page.get_static_site_paths` method (which returns a generator of URL paths for use by static site generator packages) has been removed. Packages relying on this functionality should provide their own fallback implementation.
 
+### Slug field widget required for auto-formatting
+
+The slug field JavaScript behaviour was previously attached to any field with an ID of `id_slug`, this has now changed to be any field with the appropriate HTML data attributes.
+
+If using a custom edit handler or set of panels for page models, the correct widget will now need to be used for these data attributes to be included. This widget will use the `WAGTAIL_ALLOW_UNICODE_SLUGS` Django setting.
+
+```python
+from wagtail.admin.widgets.slug import SlugInput
+# ... other imports
+
+class MyPage(Page):
+    promote_panels = [
+        FieldPanel("slug", widget=SlugInput),
+        # ... other panels
+    ]
+```
+
+Additionally, the slug behaviour can be attached to any field easily by including the following attributes in HTML or via Django's widget `attrs`.
+
+```html
+<input type="text" name="slug" data-controller="w-slug" data-action="blur->w-slug#slugify" />
+```
+
+To allow unicode values, add the data attribute value;
+
+```html
+<input type="text" name="slug" data-controller="w-slug" data-action="blur->w-slug#slugify" data-w-slug-allow-unicode="true" />
```

Comment:
I'll do a PR - this line should say

```
data-w-slug-allow-unicode-value="true" 
```

With value at the end.

Reply:
Oh right that’s good 
Good eye on noticing this😅

Reply:
PR up with a fix https://github.com/wagtail/wagtail/pull/10295